hazelcast/src/main/java/com/hazelcast/core/Cluster.java

```diff
@@ -169,6 +170,16 @@
     void changeClusterState(ClusterState newState, TransactionOptions transactionOptions);
 
     /**
+     *
+     * @return the version at which this cluster operates.
+     */
+    Version getClusterVersion();
+
+    void changeClusterVersion(Version version);
```

Comment:
Javadoc needed. And what about calling this 'setCusterVersion' ? What is the added value of calling this 'change'?